### PR TITLE
delete dead progressRedesign code

### PR DESCRIPTION
**Pull request by Bjvanminnen**
Now that the progress redesign is no longer behind an experiment flag, but enabled by default, we have a bunch of dead code that can be removed.

### CODE REVIEW

File: apps/src/templates/progress/ProgressTable.jsx [hunk 1]
```diff
@@ -31,7 +31,8 @@ const ProgressTable = React.createClass({
     // TODO - This modifies things outside of our scope. This is done right now
     // because we only want to modify this (dashboard-owned) markup in the case
     // where an experiment is enabled (leading to this component being rendered).
-    // Once we're not behind an experiment, we should make these changes elsewhere.
+    // Once we're not behind an experiment for progressRedesign, we should make
+    // these changes elsewhere.
```
Comment (Bjvanminnen) [inline apps/src/templates/progress/ProgressTable.jsx]: This is something that can still be cleaned up, but is going to be a little less straightforward, so I want to do it as a separate PR.

File: apps/src/code-studio/components/progress/CourseProgress.jsx [hunk 2]
```diff
@@ -1,24 +1,11 @@
 import React from 'react';
 import Radium from 'radium';
 import { connect } from 'react-redux';
-import _ from 'lodash';
-import { stageShape } from './types';
-import CourseProgressRow from './CourseProgressRow';
+// TODO: Delete CourseProgressRow
```
Comment (davidsbailey) [inline apps/src/code-studio/components/progress/CourseProgress.jsx]: looks done

File: apps/src/code-studio/components/progress/CourseProgress.jsx [hunk 3]
```diff
@@ -30,38 +17,21 @@ const CourseProgress = React.createClass({
     perLevelProgress: React.PropTypes.object.isRequired,
     scriptName: React.PropTypes.string.isRequired,
     professionalLearningCourse: React.PropTypes.bool,
-    focusAreaStageIds: React.PropTypes.arrayOf(React.PropTypes.number),
```
Comment (davidsbailey) [inline apps/src/code-studio/components/progress/CourseProgress.jsx:33]: this should also be removed from MiniViewTopRow.story.jsx

File: apps/src/templates/progress/ProgressTable.jsx (hunk 1 above)
Comment (davidsbailey) [reply apps/src/templates/progress/ProgressTable.jsx]: "Once we're not" --> "Now that we're not" ?

File: apps/src/code-studio/components/progress/CourseProgress.jsx (hunk 2 above)
Comment (Bjvanminnen) [reply apps/src/code-studio/components/progress/CourseProgress.jsx]: strange, caught that myself when i reviewed and thought I force pushed removing it :man_shrugging:

File: apps/src/code-studio/components/progress/MiniViewTopRow.story.jsx [hunk 4]
```diff
@@ -7,6 +7,12 @@ import progress from '@cdo/apps/code-studio/progressRedux';
 export default storybook => {
   const initialState = {
     progress: {
+      stages: [
+        {
+          levels: []
+        }
+      ],
+      focusAreaStageIds: [],
```
Comment (davidsbailey) [inline apps/src/code-studio/components/progress/MiniViewTopRow.story.jsx]: Sorry for the bad advice -- looks like i only searched `code-studio/components`.

Comment (Bjvanminnen) [reply apps/src/code-studio/components/progress/MiniViewTopRow.story.jsx]: I was also fooled. Though this particular component doesn't use this state, descendants do extract it from the store via connect.
